src/views/NewDog/NewDog.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import AddDog from '../../components/AddDog';
+import { createDog } from '../../services/fetchdogs';
+
+import './NewDog.css';
+
+export default function Admin() {
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [bio, setBio] = useState('');
+  const [breed, setBreed] = useState('');
+  const [image, setImage] = useState('');
+  const [error, setError] = useState('');
+  const history = useHistory();
+
+
+
+  const submitDog = async () => {
+    try {
+      await createDog({ name, age, bio, breed, image });
```

Comment:
It looks like name, age, bio, breed, and image are all allowed to be null on your supabase account. We can prevent this by having 'allow nullable' not checked on each column. Another way to prevent this, and also a great second check, is to have 'required' on all input fields.